virttest/qemu_devices/qdevices.py

```diff
@@ -1298,7 +1298,7 @@ class Dimm(QDevice):
     the QOM objects 'memory-backend-ram' and 'memory-backend-file'
     """
 
-    __attributes__ = ["memdev", "slot", "addr", "node"]
+    __attributes__ = ["memdev", "slot", "addr", "node", "size"]
```

Comment:
The 'size' option is not usable for dimm.

Reply:
Reverted it according to your comment.